utils/SwervePoseEstimatorBase.java

```diff
@@ -112,8 +113,8 @@ public void update() {
 
         poseEstimator.update(driveBase.getGyroscopeRotation(), driveBase.getModulePositions());
         field2d.setRobotPose(getAbsoluteFieldOrientedPoseFromAllianceOriented());
-        SmartDashboard.putString("Estimated Pose",
-                getFormattedPose(getAbsoluteFieldOrientedPoseFromAllianceOriented()));
+        Logger.getInstance().recordOutput("DriveBase/Estimated Pose",
```

Comment:
Why are we hardcoding DriveBase here whereas above we were using getName()? Same comment as elsewhere: let's make this key a constant